### PR TITLE
DEVX-238: augment microservices demo to use monitoring interceptors

### PR DESCRIPTION
**Pull request by ybyzek**
Use the new monitoring interceptor functionality provided by #153

### CODE REVIEW

**Comment by ybyzek** [comment]
Streams team: I’m particularly interested to understand why my workaround for separate processors was required (single `validations` processor caused the second one to not send monitoring interceptors)

File: src/main/java/io/confluent/examples/streams/microservices/OrdersService.java [hunk 1]
```diff
@@ -330,6 +332,7 @@ private KafkaStreams startKStreams(String bootstrapServers) {
   private Properties config(String bootstrapServers) {
     Properties props = baseStreamsConfig(bootstrapServers, "/tmp/kafka-streams", SERVICE_APP_ID);
     props.put(StreamsConfig.APPLICATION_SERVER_CONFIG, host + ":" + port);
+    MonitoringInterceptorUtils.maybeConfigureInterceptorsStreams(props);
```
Comment (mjsax) [inline src/main/java/io/confluent/examples/streams/microservices/OrdersService.java]: Is this not executed within `baseStreamsConfig` already?

**Comment by mjsax** [comment]
Build failed. Not sure why. Can you investigate @ybyzek?

**Comment by ybyzek** [comment]
I closed this because of a repeat issue whereby a rebase didn't pick up the latest 5.0.0-post changes.  There's also another issue I'm investigating, so probably more commits coming.  Closing for now, I'll open a PR when ready and ensure it builds.  Thanks @mjsax , @bbejeck

**Pull request closed without merging**